### ISSUE TITLE
Slider filter not working

### ISSUE THREAD

**Issue by woutk88** [EmicoEcommerce/Magento2Tweakwise-archived#45]
Issue Brief
--
In the Tweakwise Navigator we created a filter option 'Price' based on the property `final_price` (which is set to output Numerical). The display type for the filter option is set to Slider.

On the front-end the slider is displayed, but it's not working correctly. When trying to set/change values the filter is not being applied and we end up with a URL like this: 

> `category-x/?final_price=%7B%7Bfrom%7D%7D-%7B%7Bto%7D%7D`

 (instead of something like: `category-x/?final_price=40-120`).

Environment
--
* PHP Version: 7.0
* Magento Version: 2.2.2 (enterprise)
* Tweakwise Version: 1.3.0, 1.3.1
* Tweakwise Export Version: 1.1.2
* Magento Deploy Mode: production, development
* Third party modules: -

Steps to reproduce
--
1. Install/configure Tweakwise and export products to Tweakwise.
2. In the navigator set the feed property `final_price` to `Numerical`.
3. Create a filter template.
4. Add a new filter to the template (select `final_price`).
5. Edit the added filter and set `Display filter values` to `Slider`.
6. On the front-end visit a category and use/adjust the price slider.

Actual result
--
1. Price filter not being applied.
2. It reloads the page with this parameter added to the URL: `?final_price=%7B%7Bfrom%7D%7D-%7B%7Bto%7D%7D`.

Expected result
--
List the expected results as a bullet list of expectations
- [ ] Price filter is applied.
- [ ] Url parameter (values) are correct.

**Comment by jasperzeinstra** [comment]
This can be fixed by changing the `slider.phtml` template.

Wrapping `{{from}}` and `{{to}}` in a `urlencode` function.

```
url = url.replace('<?= urlencode('{{from}}') ?>', ui.values[0]);
url = url.replace('<?= urlencode('{{to}}') ?>', ui.values[1]);
```

**Comment by jasperzeinstra** [comment]
@Fgruntjes I'm wondering why this is still an issue in 1.3.2 and that's already reported for 1.3.0.

**Comment by EduardSelles** [comment]
@jasperzeinstra It is a priority issue. We try to take it into the development as soon as possible.

**Comment by edwinljacobs** [comment]
This has been fixed with release 1.4.0

With kind regards.
